### DISCUSSION TITLE
Media

### DISCUSSION THREAD

**Discussion by teocto** [mixcore/mix.core#682] [Q&A]
I have problems, I can't load images. They are saved but not aviable to watch them

**Comment by teocto** [comment]
![image](https://github.com/mixcore/mix.core/assets/43596506/55422481-1bd4-4686-af6f-ddc45aac91ac)
only top part is loaded

**Reply by Smilefounder** [reply]
Hi @teocto are you able to share with me the cms link to check the issue? Have you tried to upload different images yet?

**Reply by teocto** [reply]
Hi @Smilefounder . Thank you for the answer. 
I can't add repo to mixcore github.
I've just created new project and situation is the same. Db is postgres

**Reply by Smilefounder** [reply]
I cannot reproduce the same issue from my side. Are you able to deploy the site to the internet so we can take a look?

**Reply by teocto** [reply]
https://github.com/teocto/mixcore
